docs/04-operation-guides/operations/02-install-kyma.md

```diff
@@ -22,11 +22,13 @@ You can either use an out-of-the-box k3d cluster or choose any other cluster pro
   kyma provision k3d
   ```
 
-  > **TIP:** If you want to define the name of your k3d cluster and pass arguments to the Kubernetes API server (for example, to log to stderr), run:
-  >
-  > ```bash
-  > kyma provision k3d --name='{CUSTOM_NAME}' --k3s-arg='--alsologtostderr@server:0'
-  > ```
+  But you can do more. To define the name of your k3d cluster and pass arguments to the Kubernetes API server, for example, to log to stderr, run:
```

Comment:
```suggestion
  To define the name of your k3d cluster and pass arguments to the Kubernetes API server, for example, to log to stderr, run:
```
I'm not sure if we need the first sentence.

Reply:
I agree we don't strictly need it, but the whole section is so dominated by boxes (code boxes, Note box) that a little more flesh on the normal text made it easier to read for me.

<img width="762" alt="image" src="https://user-images.githubusercontent.com/76950046/156351602-3cfa9839-e0da-4ba2-9d81-b6e5b902fd02.png">

Reply:
Ok, now I see it :)